tests/python/unittest/test_operator.py

```diff
@@ -489,7 +489,9 @@ def frelu_grad(x):
     check_symbolic_backward(y, [xa], [np.ones(shape)], [ga])
 
 
-@with_seed(1234)
+# NOTE(haojin2): Fixing the seed here to deal with the precision issue with fp16. Totally
+# randomized tests may result in flaky behaviors, but most randomized tests ran locally passed.
```

Comment:
Looks like these comments need to be updated?

Reply:
Done.